lib/pyproject.toml

```diff
@@ -74,7 +74,6 @@ exclude = ["tests"]
 
 [tool.setuptools_scm]
 root = ".."
-write_to = "lib/pgfinder/_version.py"
```

Comment:
Will this still allow users to access the `__version__` variable?

I thought the purpose of writing the version to a file is so that the `importlib.metadata.version()` function can get the version, if its not written to a file where is it stored in the package and accessed from?

Reply:
I think it's somehow still being written, but I'll keep an eye on it and double-check before anything is finalized!

![image](https://github.com/Mesnage-Org/pgfinder/assets/6251883/9f8e788f-bad3-474b-b471-624aeefb0ff5)

Reply:
Not 110% sure how, but the versioning does still work, and it broke the CI when it was trying to write that file! It also works in the GUI (which reads the `__version__` variable)!
![image](https://github.com/Mesnage-Org/pgfinder/assets/6251883/38fe5509-46d6-456d-bb83-ad1fcb3391fd)

Reply:
Its :mage: ! 

Ok as long as the versioning is still available I'm not too concerned, its just that its critical to know what version errors occur with when trouble shooting.

Thanks for checking :+1: